pkg/console/operator/sync_v400.go

```diff
@@ -240,12 +240,11 @@ func SyncSecret(co *consoleOperator, recorder events.Recorder, operatorConfig *o
 // therefore no additional error handling is needed here.
 func SyncConfigMap(co *consoleOperator, recorder events.Recorder, operatorConfig *operatorv1.Console, consoleConfig *configv1.Console, infrastructureConfig *configv1.Infrastructure, rt *routev1.Route) (*corev1.ConfigMap, bool, error) {
 	logrus.Printf("validating console configmap...")
-
-	defaultConfigmap, _, err := configmapsub.DefaultConfigMap(operatorConfig, consoleConfig, infrastructureConfig, rt)
+	managedConfig, err := co.configMapClient.ConfigMaps(api.OpenShiftConsoleOperatorNamespace).Get(api.OpenshiftManagedConfigMap, metav1.GetOptions{})
```

Comment:
Wondering if we should do the `.Get()` where we get the other configs:
https://github.com/openshift/console-operator/blob/master/pkg/console/operator/operator.go#L151

Its a lot of configs to plumb through... but prob reveals that some tech debt to clean-up later would be to collect them and pass them 

```golang
// in pkg/console/operator/operator.go#L151
// since we have to pass them all so much?
type ConfigSet struct {
   consoleConfig
   operatorConfig
   infrastructureConfig
   ingressConfig
   brandingConfig
}
```

Reply:
I originally put it at the very top (func sync) and passed it all the way through, but 1) it was not needed in other functions and 2) it was accessible through the consoleOperator.
My follow up question would be do the other items need to be passed?